### PR TITLE
LPS-100534 | Apply new Session utilities in portal-web

### PR DESCRIPTION
**Pull request by mateomustapic**

### CODE REVIEW

**Comment by mateomustapic** [comment]
Hi @jbalsas ,

I tested all the changes in this pr except the ones in this file
`https://github.com/jbalsas/liferay-portal/blob/a2ac68ca8a90421891db1a9e4bb77bf8cf058d1c/portal-web/docroot/html/taglib/ui/form_navigator/lexicon/page.jsp`
I wasn't able to find usage of this code in the portal.
Can please check if changes in this file are okay?

File: portal-web/docroot/html/common/themes/user_locale_options.jsp [hunk 1]
```diff
@@ -62,13 +62,13 @@ String currentURL = PortalUtil.getCurrentURL(request);
...
-	<aui:script use="aui-base,liferay-store">
+	<aui:script use="aui-base">
 		var ignoreUserLocaleOptionsNode = A.one('#ignoreUserLocaleOptions');
 
 		ignoreUserLocaleOptionsNode.on(
 			'click',
 			function() {
-				Liferay.Store(
+				Liferay.Util.Session.set(
 					{
```
Comment (jbalsas) [inline portal-web/docroot/html/common/themes/user_locale_options.jsp]: Is this actually working? If I remember correctly, we didn't implement the `setAll`, so we should probably loop over the keys here or just have 2 different set calls.

File: portal-web/docroot/html/taglib/ui/toggle/page.jsp [hunk 2]
```diff
@@ -40,69 +40,64 @@ String defaultMessage = (String)request.getAttribute("liferay-ui:toggle:defaultM
 <aui:script>
 	var <%= stateVar %> = '<%= defaultStateValue %>';
 
-	Liferay.provide(
-		window,
-		'<%= stateVar %>Toggle',
-		function(state, saveState) {
-			var A = AUI();
+	window<%= stateVar %>Toggle = function(state, saveState) , function(state, saveState) {
```
Comment (jbalsas) [inline portal-web/docroot/html/taglib/ui/toggle/page.jsp]: This doesn't look like valid js syntax:

```
window.somethingToggle = function(state, saveState) , function(state, saveState) {
```

File: portal-web/docroot/html/common/themes/user_locale_options.jsp (hunk 1 above)
Comment (mateomustapic) [reply portal-web/docroot/html/common/themes/user_locale_options.jsp]: No, this was my leftover code. I will force push this and the change bellow

File: portal-web/docroot/html/taglib/ui/toggle/page.jsp [hunk 3]
```diff
@@ -40,69 +40,64 @@ String defaultMessage = (String)request.getAttribute("liferay-ui:toggle:defaultM
 <aui:script>
 	var <%= stateVar %> = '<%= defaultStateValue %>';
 
-	Liferay.provide(
-		window,
-		'<%= stateVar %>Toggle',
-		function(state, saveState) {
-			var A = AUI();
+	window <%= stateVar %>Toggle = function(state, saveState) {
```
Comment (jbalsas) [inline portal-web/docroot/html/taglib/ui/toggle/page.jsp]: Hey @mateomustapic, this still seems wrong. I assume this should be `window.<%= stateVar %>Toggle`

File: portal-web/docroot/html/common/themes/user_locale_options.jsp [hunk 4]
```diff
@@ -68,12 +68,9 @@ String currentURL = PortalUtil.getCurrentURL(request);
 		ignoreUserLocaleOptionsNode.on(
 			'click',
 			function() {
-				Liferay.Store(
-					{
-						ignoreUserLocaleOptions: true,
-						useHttpSession: true
-					}
-				);
+				Object.entries({ignoreUserLocaleOptions: true, useHttpSession: true}).forEach((key, value) => {
```
Comment (jbalsas) [inline portal-web/docroot/html/common/themes/user_locale_options.jsp]: This seems a bit unnecessary, why not simply doing:

```
Liferay.Util.Session.set('ignoreUserLocaleOptions', true);
Liferay.Util.Session.set('useHttpSession', true);
```

Comment (mateomustapic) [reply portal-web/docroot/html/common/themes/user_locale_options.jsp]: I will simplify and resend in few minutes

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/78227. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/78228. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/acd4f7dece07~1...acd4f7dece07) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**